python/kvikio/_lib/pynvcomp.pyx

```diff
@@ -252,9 +237,9 @@ cdef class _LZ4Decompressor:
         del self.d
 
     cpdef configure(self, in_arr, in_bytes, temp_bytes, out_bytes, uintptr_t stream=0):
```

Comment:
```suggestion
    cpdef configure(self, in, temp_bytes, out_bytes, uintptr_t stream=0):
```

Reply:
This existing signature matches nvcomp 2.1, I wouldn't want to make them out of sync. However, Nvcomp 2.2 is considerably different.

Reply:
Gotcha. Perhaps this isn't where things would be coerced to `Array`. Instead maybe that happens in a Python layer then we pass the pointer and size from that layer to here